### PR TITLE
test: [M3-7395] Add new cypress tests to longview landing page

### PR DESCRIPTION
**Pull request by cliu-akamai**
## Description 📝
Add new tests to check the empty state message in Longview landing page.

## Major Changes 🔄
- Add new cypress tests to tests to check Longview empty state

## How to test 🧪
```
yarn cy:run -s "cypress/e2e/core/longview/longview.spec.ts"
```

### CODE REVIEW

File: packages/manager/cypress/e2e/core/longview/longview.spec.ts [hunk 1]
```diff
@@ -103,7 +111,7 @@ describe('longview', () => {
    * - Creates a Linode, connects to it via SSH, and installs Longview using the given cURL command.
    * - Confirms that Cloud Manager UI updates to reflect Longview installation and data.
    */
-  it('can install Longview client on a Linode', () => {
+  it.skip('can install Longview client on a Linode', () => {
```
Comment (jdamore-linode) [inline packages/manager/cypress/e2e/core/longview/longview.spec.ts:114]:
```suggestion
  it('can install Longview client on a Linode', () => {
```

File: packages/manager/cypress/e2e/core/longview/longview.spec.ts [hunk 2]
```diff
@@ -176,4 +184,44 @@ describe('longview', () => {
...
+
+    ui.button
+      .findByTitle(longviewAddClientButtonText)
+      .should('be.visible')
+      .should('be.enabled')
+      .click();
+    cy.wait('@ceateLongviewClient');
+
+    // Confirms that UI updates to show the new client when creating one.
+    cy.findByText(`longview-client-${client.id}`).should('be.visible');
```
Comment (jdamore-linode) [inline packages/manager/cypress/e2e/core/longview/longview.spec.ts:214]:
```suggestion
    cy.findByText(`${client.label}`).should('be.visible');
```
Very minor change, but this way if the Longview client factory label ever changes the test will still pass 👍

**Review by jdamore-linode — APPROVED** [review]
Great work @cliu-akamai, really nicely done! Approved pending removal of that stray `.skip()`

File: packages/manager/.changeset/pr-10321-tests-1712172117252.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"@linode/manager": Tests
+---
+
+Add new cypress tests to longview landing page ([#10321](https://github.com/linode/manager/pull/10321))
```
Comment (dwiley-akamai) [inline packages/manager/.changeset/pr-10321-tests-1712172117252.md:5]:
```suggestion
Add new Cypress tests for Longview landing page ([#10321](https://github.com/linode/manager/pull/10321))
```

**Review by dwiley-akamai — APPROVED** [review]
Tests pass ✅ 
Code review ✅

File: packages/manager/cypress/e2e/core/longview/longview.spec.ts [hunk 4]
```diff
@@ -176,4 +184,44 @@ describe('longview', () => {
...
+
+  /*
+   * - Confirms that the landing page empty state message is displayed when no Longview clients are present.
+   * - Confirms that UI updates to show the new client when creating one.
+   */
+  it('displays empty state message when no clients are present and shows the new client when creating one', () => {
+    const client: LongviewClient = longviewClientFactory.build();
+    const status: LongviewResponse = longviewResponseFactory.build();
+    mockGetLongviewClients([]).as('getLongviewClients');
+    mockCreateLongviewClient(client).as('ceateLongviewClient');
```
Comment (hkhalil-akamai) [inline packages/manager/cypress/e2e/core/longview/longview.spec.ts:196]:
```suggestion
    mockCreateLongviewClient(client).as('createLongviewClient');
```

File: packages/manager/cypress/e2e/core/longview/longview.spec.ts [hunk 5]
```diff
@@ -176,4 +184,44 @@ describe('longview', () => {
...
+    // Confirms that a landing page empty state message is displayed
+    cy.findByText(longviewEmptyStateMessage).should('be.visible');
+    cy.findByText(longviewAddClientButtonText).should('be.visible');
+
+    ui.button
+      .findByTitle(longviewAddClientButtonText)
+      .should('be.visible')
+      .should('be.enabled')
+      .click();
+    cy.wait('@ceateLongviewClient');
```
Comment (hkhalil-akamai) [inline packages/manager/cypress/e2e/core/longview/longview.spec.ts:211]:
```suggestion
    cy.wait('@createLongviewClient');
```

**Review by hkhalil-akamai — APPROVED** [review]
Nice work!
